Makefile

```diff
@@ -38,7 +38,17 @@ guides:	swagger ## build the developer guide docs
 	cd docs/guides; make clean html
 
 extract_translations: ## extract localizable strings from sources
-	i18n_tool extract -v
+	# (IS_OPENEDX_TRANSLATIONS_WORKFLOW) is set to "yes" when openedx-translations calls this target.
+	# Otherwise, we preserve the old behavior of having segmented po files.
+	if [ "$(IS_OPENEDX_TRANSLATIONS_WORKFLOW)" = "yes" ]; then \
```

Comment:
I recommend using `ifeq` Makefile to ensure it's compatible across operating systems: [example](https://github.com/openedx/frontend-app-gradebook/blob/1c704585908d21aa4ccee40affb29a086f29067a/Makefile#L58-L75).

It also makes the `Makefile` more readable because less `/` slashes are appended to the script.

Reply:
done